.circleci/config.yml

```diff
@@ -78,6 +98,16 @@ jobs:
       FIREFOX_VERSION: :latest
     steps:
       - checkout
+      - run:
+          name: Check file paths
+          command: |
+            if git diff --name-only main HEAD | grep -E 'app/experimenter/integration'
```

Comment:
1) The path `app/experimenter/integration` doesn't exist?
2) My point was that we can not gate this circle job on any path until we more clearly separate it into more specific groups of tests

Reply:
oooooooooh okay, so we run these everytime